jujubigdata/templates/systemd.conf

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description={{service}} server
+After=network.target
+After=systemd-user-sessions.service
+
+[Service]
+User={{user}}
+Type=forking
+EnvironmentFile=-/etc/environment
+ExecStart={{hadoop_path}}/sbin/{{daemon}}-daemon.sh --config {{hadoop_conf}} start {{service}}
+ExecStop={{hadoop_path}}/sbin/{{daemon}}-daemon.sh --config {{hadoop_conf}} stop {{service}}
```

Comment:
I think we need to add a restart option here.  Otherwise, if the service dies, it stays dead.  Suggest adding this to the [Service] block:

`Restart:on-failure`